packages/addresses/src/__tests__/program-derived-address-test.ts

```diff
@@ -0,0 +1,141 @@
+                seeds: [new Uint8Array([1, 2, 3])],
+            })
+        ).resolves.toStrictEqual({
+            bumpSeed: 255,
+            pda: '9Tj3hpMWacDiZoBe94sjwJQ72zsUVvEQYsrqyy2CfHky',
+        });
+    });
+    it('returns a program derived address after having tried multiple bump seeds given a program address and a byte-array seed', async () => {
+        expect.assertions(1);
+        await expect(
+            getProgramDerivedAddress({
+                programAddress: '9HT3iB4oX1aZPH5V8eNUGByKuwhfcKjBQ3x9rfEAuNeF' as Base58EncodedAddress,
+                seeds: [new Uint8Array([1, 2, 3])],
+            })
+        ).resolves.toStrictEqual({
+            bumpSeed: 251,
+            pda: 'EeTcRajHcPh74C5D4GqZePac1wYB7Dj9ChTaNHaTH77V',
+        });
+    });
+    it('returns a program derived address given a program address and a string seed', async () => {
+        expect.assertions(1);
+        await expect(
+            getProgramDerivedAddress({
+                programAddress: 'EKaNRGA37uiGRyRPMap5EZg9cmbT5mt7KWrGwKwAQ3rK' as Base58EncodedAddress,
+                seeds: ['hello'],
+            })
+        ).resolves.toStrictEqual({
+            bumpSeed: 255,
+            pda: '6V76gtKMCmVVjrx4sxR9uB868HtZbL3piKEmadC7rSgf',
+        });
+    });
+    it('returns a program derived address after having tried multiple bump seeds given a program address and a string seed', async () => {
+        expect.assertions(1);
+        await expect(
+            getProgramDerivedAddress({
+                programAddress: '9PyoV2rqNtoboSvg2JD7GWhM5RQvHGwgdDvK7MCfpgX1' as Base58EncodedAddress,
+                seeds: ['hello'],
+            })
+        ).resolves.toStrictEqual({
+            bumpSeed: 251,
+            pda: 'E6npEurFu1UEbQFh1DsqBvny17XxUK2QPMgxD3Edn3aG',
+        });
+    });
+    it('returns a program derived address given a program address and a UTF-8 string seed', async () => {
+        expect.assertions(1);
+        await expect(
+            getProgramDerivedAddress({
+                programAddress: 'A5dcVPLJsE2vbf7hkqqyYkYDK9UjUfNxuwGtWF2m2vEz' as Base58EncodedAddress,
+                seeds: ['\uD83D\uDE80'],
+            })
+        ).resolves.toStrictEqual({
+            bumpSeed: 255,
+            pda: 'GYpAzW57Ex4Sw3rp4pq95QrjvtsDyqZsMhSZwqz3NMsE',
+        });
+    });
+    it('returns a program derived address after having tried multiple bump seeds given a program address and a UTF-8 string seed', async () => {
+        expect.assertions(1);
+        await expect(
+            getProgramDerivedAddress({
+                programAddress: 'H8gBP21L5ietkHgXcGbgQBCVVEdPUQyuP9Q5MPRLLSJu' as Base58EncodedAddress,
+                seeds: ['\uD83D\uDE80'],
+            })
+        ).resolves.toStrictEqual({
+            bumpSeed: 251,
+            pda: '46v3JvPtEPeQmH3euXydEbxYD6yfxeZjWSzkkYvvM5Pp',
+        });
+    });
```

Comment:
Just out of interest did you use some trick to generate these examples? Mainly asking because they all have bumpSeed 251

Reply:
Generating a few addresses for the tests until exactly that is the case.